core/interface.go

```diff
@@ -210,7 +212,7 @@ type BlockContext struct {
 	Coinbase    Address
 	GasLimit    uint64
 	BlockNumber *big.Int
-	Time        *big.Int
+	Time        uint64
```

Comment:
I'm assuming this change corresponds to a similar change in Foundation Geth.

I'd be inclined to leave Time as a *big.Int and convert from the uint64 to a big.Int (eg. `new(big.Int).SetInt64(int64(ctx.Time))`) so that plugins don't have to update their interaction with BlockContext. Otherwise we need to update both ETC Geth and Bor to translate the other way, and `*big.Int -> uint64` is more error prone than `uint64 -> *big.Int`